### DISCUSSION TITLE
File structure

### DISCUSSION THREAD

**Discussion by zithith** [stanavdb/cosmere-rpg#39] [General]
Let's discuss the folder structure we've got.
On the whole, no issues. It matches common Foundry practice in how everything is broken up.
I just want to start getting some things in writing  so eventually some of it can be copy/pasted into a codified guide (.md file/other doc format) in the repo so everyone is on the same page.

First thing I notice is: consistency in folder naming: when are we using plurals and when singular?

**Comment by stanavdb** [comment]
My initial thoughts on plural/singular: 
Any folder that contains a set of things (e.g. applications) should be plural. Any folder that contains a single unit of code (i.e. a module) should be singular. With the exception being the Document types (Item, Actor, etc.) to be consistent with Foundry common practice. (And use common sense anywhere those rules don't apply.)

That's just what comes to mind, I'm open to better ideas
